lib/iris/tests/unit/lazy_data/test_map_complete_blocks.py

```diff
@@ -134,3 +135,24 @@ def test_multidimensional_input(self):
         )
         assert is_lazy_data(result)
         assert_array_equal(result.compute(), array + 1)
+
+    def test_rechunking(self):
+        # Choose a dask array with an irregularly chunked dimension to be rechunked.
+        lazy_array = da.ones((5, 9, 10, 10), chunks=(2, 5, 10, 5))
+        cube, _ = create_mock_cube(lazy_array)
+
+        result = map_complete_blocks(
+            cube, self.func, dims=(2, 3), out_sizes=(30, 40), dtype=lazy_array.dtype
+        )
+        assert is_lazy_data(result)
```

Comment:
I don't think this bit is contributing anything much, as we don't check any properties of the result, except that it is lazy.
I think we can remove this, and only check the result with a restricted chunksize.
```suggestion
#(nothing)
```